install_config/configuring_nuagesdn.adoc

```diff
@@ -0,0 +1,112 @@
+[[install-config-configuring-nuage-sdn]]
+= Configuring Nuage SDN
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Installation
+
+The VSP integration with {product-title} works for both virtual machines (VMs) and baremetal installations of {product-title}.
+
+Ansible is the preferred method of installing {product-title} with Nuage VSP as the network plugin. Refer to the xref:index.adoc#install-config-index[{product-title} installation and configuration section] for details on how to install {product-title} using Ansible.
+
+=== Installation for single master
+
+In the Ansible nodes file, the following parameters need to be specified in order to setup Nuage VSP as the network plugin.
+
+ # Nuage specific parameters
+ openshift_use_openshift_sdn=False
+ openshift_use_nuage=True
+ os_sdn_network_plugin_name='nuage/vsp-openshift'
+ openshift_node_proxy_mode='userspace'
+ nuage_openshift_monitor_rest_server_port=9443
+
+ # VSP related parameters
+ vsd_api_url=https://192.168.103.200:8443
+ vsp_version=v4_0
+ enterprise=nuage
+ domain=openshift
+ vsc_active_ip=192.168.103.201
+ vsc_standby_ip=192.168.103.202
+ uplink_interface=eth0
+
+ # rpm locations
```

Comment:
If Red Hat is supporting this plugin the 'location_of_rpm_server' should come from the Red Hat CDN, and this should be a channel based option. What channel will be providing these plugins?